vmm/src/cpu.rs

```diff
@@ -379,14 +376,31 @@ impl Vcpu {
         })))
     }
 
+    #[cfg(target_arch = "aarch64")]
+    /// Configures a aarch64 specific vcpu and should be called once per vcpu when created.
+    ///
+    /// # Arguments
+    ///
+    /// * `machine_config` - Specifies necessary info used for the CPUID configuration.
+    /// * `kernel_entry_point` - Kernel entry point address in guest memory and boot protocol used.
+    /// * `vm_memory` - Guest memory.
+    pub fn configure(
+        &mut self,
+        #[allow(unused_variables)] kernel_entry_point: Option<EntryPoint>,
+        #[allow(unused_variables)] vm_memory: &GuestMemoryAtomic<GuestMemoryMmap>,
```

Comment:
Could you please use `_kernel_entry_point` and `_vm_memory` to avoid the usage of `#[allow(unused_variables)]`